partner-portal/src/API/Controllers/CallbackController.cs

```diff
@@ -63,21 +63,21 @@ public async Task<IActionResult> Create([FromBody] Callback callback)
 
 
         [HttpGet("driver")]
-        [ProducesResponseType(typeof(IEnumerable<ViewModels.Callback>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<ViewModels.CaseCallback>), 200)]
```

Comment:
add a TODO on line 36 to replace proto model with a request model e.g. "CallbackRequest"

Reply:
Added a comment to fix this